.goreleaser.yaml

```diff
@@ -1,7 +1,8 @@
 version: 2
+project_name: test-engine-client
```

Comment:
Add `project_name` here as I found the binary being put into `dist/test-splitter_*` on the test engine client pipeline build. When testing locally it puts the binary to `dist/test-engine-client*`


> building binaries
> --
>   | 2024-09-16 16:51:08 ACST | • partial build                                  match=target=darwin_amd64
>   | 2024-09-16 16:51:08 ACST | • building                                       binary=dist/test-splitter_darwin_amd64_v1/bktec

The example build: https://buildkite.com/buildkite/test-engine-client/builds/1003#0191f9b3-4701-4d79-951a-7c8a44704f89


@nprizal is this a sensible change?

Reply:
Ah, interesting! g
oreleaser get the project name from repository name by default, we have renamed the repository, but the pipeline still referring to the old repo name (see image below). I think explicitly setting the project name is a good idea here.
<img width="866" alt="Screenshot 2024-09-17 at 9 14 26 AM" src="https://github.com/user-attachments/assets/41e98e78-2dcc-4447-95db-d53e0a243b5c">